plug.vim

```diff
@@ -2028,10 +2067,11 @@ function! s:git_validate(spec, check_branch)
               \ branch, a:spec.branch)
       endif
       if empty(err)
-        let [ahead, behind] = split(s:lastline(s:system(printf(
+        let [output, shellerror] = s:system_with_error(printf(
               \ 'git rev-list --count --left-right HEAD...origin/%s',
-              \ a:spec.branch), a:spec.dir)), '\t')
-        if !v:shell_error && ahead
+              \ a:spec.branch), a:spec.dir)
+        let [ahead, behind] = split(s:lastline(output), '\t')
```

Comment:
Hmm, this can fail if `output` does not have `\t` because of an error, right? It was my oversight.